js/app/src/main.ts

```diff
@@ -79,11 +79,11 @@ function create_custom_element() {
 					info: info === "false" ? false : true,
 					container: container === "false" ? false : true,
 					is_embed: is_embed === "false" ? false : true,
-					initial_height: initial_height ?? undefined,
+					initial_height,
```

Comment:
`initial_height` is already ensured to be non-null at https://github.com/gradio-app/gradio/pull/4223/files#diff-61bce78d1e96b6919fb28c4d65408d0a9299a0e612aa75cbadcefe0f0bf7c14dR63